components/sandbox/README.md

```diff
@@ -0,0 +1,14 @@
+This component is used to embed embeds!
```

Comment:
Well, more precisely, this component has nothing to do with embeds specifically, it just provides an isolated environment for any arbitrary HTML via iframes.